### PR TITLE
set the implicit width and height to 32

### PR DESCRIPTION
**Pull request by mrwillyees**
fix to default height and width were 0. set to 32
it might be too small 32x32
image

### CODE REVIEW

**Comment by anmacdonald** [comment]
I see what you mean about the size being too small. My bad for picking the numbers out of a hat.

Looking at the actual image size it is 48x48: https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/main/uitools/images/compass.png

I think that would be more sensible?

**Comment by mrwillyees** [comment]
changed to 48x48 as original png
image

File: uitools/import/Esri/ArcGISRuntime/Toolkit/NorthArrow.qml [hunk 1]
```diff
@@ -27,6 +27,7 @@ import QtQuick 2.12
    heading of the SceneView. 
    Double clicking on the NorthArrow triggers the heading of the connected
    GeoView to be orientainted to 0.
+   \note default width and height is 32.
```
Comment (anmacdonald) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/NorthArrow.qml:30]: ❌  Remember to update the documentation too!

**Review by anmacdonald — CHANGES_REQUESTED** [review]
👍  Good stuff

Please test this change with both a Qt Quick C++ App, and a QML App to confirm.

After updating the documentation, if both apps work, feel free to merge using the `squash and merge` option. No need to re-review.

**Pull request merged**